pipeline.conf

```diff
@@ -0,0 +1,7 @@
+tags: [zephyr,nrf52840,cortex-m]
+name: "Zephyr on nRF52840"
+description: "[Zephyr](https://www.zephyrproject.org) shell running in Renode on Cortex-M [nRF52840](https://www.nordicsemi.com/Products/nRF52840). Full test sources available at [GitHub](https://github.com/antmicro/renode-zephyr-nrf52840)."
```

Comment:
`[Zephyr](...) Shell and Hello World samples running in Renode...`

Reply:
Done.